pages/tutorials/github_merge_queue.md

```diff
@@ -24,6 +30,4 @@ To enable a merge queue for a pipeline:
     gh-readonly-queue/{base_branch}/*
     ```
```

Comment:
If using my suggestion to keep one list, you'll need to update the indentation here:

````suggestion
        gh-readonly-queue/{base_branch}/*
        ```
````

Note, the line above also needs updating to match, but I can't comment on it.